### DISCUSSION TITLE
How to display terminal window?

### DISCUSSION THREAD

**Discussion by gavincrown** [Taitava/obsidian-shellcommands#251] [Q&A] [answered]
Obsidian v0.15.9
Shell commands v0.14.0
 
I need to press Ctrl + C to stop this command in terminal window
![image](https://user-images.githubusercontent.com/110040980/184579489-f8487379-a40b-48d3-b147-a4325b6a38eb.png)

**Comment by Taitava** [comment, marked as answer]
SC does not have an actual feature for displaying the terminal window. Now that I think of it, I'd actually like to have it, but I don't know how to implement it. I'd like to have some kind of toggleable setting _Display console window_, but I'm not sure how to implement it in Node.js's `child_process`.

## A workaround

You can probably try this kind of ugly workaround that works in Windows:
As your shell command is `hexo s`, try to change it to: `start powershell "-NoExit -Command hexo s"`

Explanation: `start powershell` opens up a PowerShell terminal window. `"-NoExit -Command hexo s"` tells the terminal window not to close after executing the command (maybe not needed in your case, because your command seems to start a process that does not end before pressing `Ctrl`+`C`) and it also passes the shell command that you want to execute in the opened terminal window.

Please let me know if this is of any help. 🙂

**Reply by gavincrown** [reply]
It works. Thank you very much!

**Reply by Taitava** [reply]
You are welcome! 🌞
